docs/src/userguide/migrating_to_cy30.rst

```diff
@@ -239,3 +239,16 @@ When running into an error it is required to add the corresponding operator::
         Example operator++(int)
         Example operator--(int)
 
+
+``**`` power operator
+=====================
+
+Cython 3 has changed the behaviour of the power operator to be
+more like Python. The consequences are that
+
+#. ``a**b`` of two ints may return a floating point type,
+#. ``a**b`` of one or more non-complex floating point numbers may end
+   up return a complex number.
```

Comment:
```suggestion
#. ``a**b`` of one or more non-complex floating point numbers may
   return a complex number.
```

Will push this change next time - don't think it justifies rerunning the CI on its own though